tasks/mainnet-manifest.json

```diff
@@ -1,8 +1,8 @@
 {
   "Pause": false,
-  "ProtocolVersion": 5,
+  "ProtocolVersion": 7,
   "InitialInstance": 0,
-  "BootstrapEpoch": 5000000,
+  "BootstrapEpoch": 4920480,
```

Comment:
Confirming that Epoch 4920480, maps to the suggested `2025-04-29T10:00:00Z`. See here for [other timezones](https://www.worldtimebuddy.com/?qm=1&lid=100,1816670,5368361,5128581&h=100&date=2025-4-30&sln=10-11&hf=undefined&c=1638): 

<img width="597" alt="Screenshot 2025-04-24 at 14 11 35" src="https://github.com/user-attachments/assets/023fb788-d4cd-46dd-a201-e05874edb510" />